src/utils/emails.js

```diff
@@ -64,10 +91,11 @@ const sendEmail = async (email, name, heading, res, messageTemplate, token) => {
     await sgMail.send(message);
     // RETURN SUCCESS MESSAGE
     /* eslint-disable no-console */
-    console.log(`Email sent to ${name}`);
+    console.log(message);
   } catch (error) {
+    console.log(error);
     return res.status(500).json({
```

Comment:
I would also suggest to not return a status when an email was not sent because it crushes the whole application when an email could not be delivered. Examples are when our SendGrid daily credits are exhausted. Instead, you can just log on console for now.